app/src/main/java/me/hufman/androidautoidrive/carapp/music/MusicApp.kt

```diff
@@ -36,6 +36,7 @@ class MusicApp(val securityAccess: SecurityAccess, val carAppAssets: CarAppResou
 	var appListViewVisible = false
 	var playbackViewVisible = false
 	var enqueuedViewVisible = false
+	var browseViewVisible = false
```

Comment:
This is fine for following the pattern, but I should change this (to match a pattern I've developed in Notifications) where each view has its own visible flag, instead of tracking these flags in this parent object. No need to include that change in this PR though, just musings :)

Reply:
I do like that new structure! It sounds like eventually going to need a PR to sync all the states to use a similar system :)